app/Traits/ImportToRogue.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Chompy\Traits;
+
+use League\Csv\Reader;
+use Chompy\Events\LogProgress;
+use Illuminate\Support\Facades\Storage;
+
+trait ImportToRogue
```

Comment:
@katiecrane do we have to add this in the `use` statement [here](https://github.com/DoSomething/chompy/pull/32/files#diff-ed45aaf47afdecacbc92e2d0393dc00bR19)?

Reply:
sorry disregard! The changes were hiding there and saw you already did that ! :)

Reply:
I'm having trouble finding this too! Where was the trait being used?